source/SpinalHDL/Simulation/bootstraps.rst

```diff
@@ -87,13 +87,22 @@ On this ``SimCompiled`` instance you can run your simulation with the following
 
 ``doSimUntilVoid[(simName[, seed])]{dut => ...}``
   Run the simulation until it is ended by calling either ``simSuccess()`` or ``simFailure()``.
-  The main stimulus thread can continue or exit early, as long as there are events to process,
+  The main stimulus thread can continue or exit early. As long as there are events to process,
   the simulation will continue. The simulation will report an error if it gets fully stuck.
 
-When in doubt about what to use, prefer ``doSim(...)``, for example :
+The following testbench template will use the following toplevel : 
 
 .. code-block:: scala
+    
+   class TopLevel extends Component {
+      val counter = out(Reg(UInt(8 bits)) init (0))
+      counter := counter + 1
+   }
+
+Here is a template with many simulation configuration :
```

Comment:
Typos

```
Here is a template with many simulation configurations:
```

Reply:
Hmm i would say fondamentaly, doSimUntilVoid name do not specify anything about what hapen once void is here.

Typo fixed ^^

Reply:
> fondamentaly, `doSimUntilVoid` name do not specify anything about what hapen once void is here.

You got the point :smile: